yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -569,8 +569,10 @@ func checkIfEndCommandCanBePerformed(msr *metadb.MigrationStatusRecord) {
 func stopVoyagerCommands(msr *metadb.MigrationStatusRecord, lockFiles []*lockfile.Lockfile) {
 	if msr.ArchivingEnabled {
 		exportDataLockFile := getLockFileForCommand(lockFiles, "export data")
+		exportDataFromTargetLockFile := getLockFileForCommand(lockFiles, "export data from target")
```

Comment:
From the looks of it, we only stop "export data" or "export data from target", but there is also "export data from source". (which is the same as "export data" but different from a command perspective.